modules/security-context-constraints-about.adoc

```diff
@@ -366,16 +366,15 @@ pod to fail.
 [id="scc-prioritization_{context}"]
 == Security context constraints prioritization
 
-Security context constraints (SCCs) have a priority field that affects the ordering when attempting to
-validate a request by the admission controller. A higher priority
-SCC is moved to the front of the set when sorting. When the complete set
-of available SCCs are determined they are ordered by:
+Security context constraints (SCCs) have a priority field that affects the ordering when attempting to validate a request by the admission controller. A priority value of `0` is the lowest possible priority. A nil priority is considered a `0`, or lowest, priority. Higher priority SCCs are moved to the front of the set when sorting.
```

Comment:
Should we move the priority explanations to a separate paragraph for clarity?

Reply:
@stlaz Can you be more specific about what you mean? This whole paragraph is about explaining priority.

Reply:
I meant having the part
```
A priority value of `0` is the lowest possible priority. A nil priority is considered a `0`, or lowest, priority. Higher priority SCCs are moved to the front of the set when sorting.
```
in a separate paragraph just like this